src/api/ZoweExplorerApiRegister.ts

```diff
@@ -95,6 +96,8 @@ export class ZoweExplorerApiRegister implements ZoweExplorerApi.IApiRegisterClie
     public registerUssApi(ussApi: ZoweExplorerApi.IUss): void {
         if (ussApi && ussApi.getProfileTypeName()) {
             this.ussApiImplementations.set(ussApi.getProfileTypeName(), ussApi);
+            // reload the internal profiles cache to add the newly added profiles
+            Profiles?.getInstance()?.refresh();
```

Comment:
Is the usage of `?` as a safe navigation operator desired here, or should an error be raised if the call to `Profiles.getInstance` fails?

Reply:
@tjohnsonBCM thanks for the review. No, the constructor of this singleton will call these method before the Profiles singleton is initialized. In that case no refresh is needed, yet. So only if Profiles already exist, after activation finished, this is needed to be called.